open-api/rest-catalog-open-api.yaml

```diff
@@ -4744,6 +4764,16 @@ components:
           }
         }
 
+    NamespaceNotEmptyError:
+      summary: The requested namespace is not empty
+      value: {
+        "error": {
+          "message": "The given namespace is not empty",
```

Comment:
nit: Summary says requested, the message says given. Just bugs me that they are different :)

Reply:
I particularly aligned this with other examples that also use "requested" in the summary and "given" in the message